test-utils/testlib/src/lib.rs

```diff
@@ -11,6 +11,9 @@ use near_store::test_utils::create_test_store;
 use std::sync::Arc;
 use tempdir::TempDir;
 
+#[macro_use]
+extern crate assert_matches;
```

Comment:
Rust 2018 edition deprecated `extern crate` with simple `use` statements (you just import the macro in the module where you used it like this: `use assert_matches::assert_matches;`)